build/build.ps1

```diff
@@ -220,6 +225,20 @@ function Build {
     $env:DOTNET_SDK_TEST_MSBUILD_PATH = Join-Path $env:VSInstallDir "MSBuild\15.0\Bin\msbuild.exe"
   }
 
+  if ($dogfood)
+  {
+    if ($properties -eq $null -and $env:DOTNET_SDK_DOGFOOD_SHELL -ne $null)
+    {
+      $properties = , $env:DOTNET_SDK_DOGFOOD_SHELL
+    }
+    if ($properties -ne $null)
+    {
+      $Host.UI.RawUI.WindowTitle = "SDK Test ($RepoRoot) ($configuration)"
+      & $properties[0] $properties[1..($properties.Length-1)]
+    }
+    exit 0
```

Comment:
Should the sh have the same early return here?

Reply:
I've added this now.  Turns out previously sourcing the script didn't work because it had an exit at the end.  I've disabled that if you pass `--dogfood`.